docs/setup/operation/configuration.md

```diff
@@ -24,12 +24,14 @@ limitations under the License.
 <div id="toc"></div>
 
 ## Zeppelin Properties
-There are two locations you can configure Apache Zeppelin.
 
-* **Environment variables** can be defined `conf/zeppelin-env.sh`(`conf\zeppelin-env.cmd` for Windows).
-* **Java properties** can be defined in `conf/zeppelin-site.xml`.
+Zeppelin can be configured via several sources.
+
+Sources descending by priority:
+ - system properties
+ - environment variables can be defined `conf/zeppelin-env.sh`(`conf\zeppelin-env.cmd` for Windows).
+ - configuration file can be defined in `conf/zeppelin-site.xml`
```

Comment:
The order should be:
* environment variable
* system property
* property in zeppelin-site.xml
![image](https://user-images.githubusercontent.com/164491/105044427-adf9dd00-5aa1-11eb-87a0-1d3d18a165b9.png)

Reply:
You are right. I will change the order to prevent breaking changes.

Reply:
Fixed.